### ISSUE TITLE
Crash on startup

### ISSUE THREAD

**Issue by swoolcock** [CelestialCartographers/Loenn#8]
Crashes both with the .app and the .love, same error.

macOS Big Sur 11.6.1

```
Error

utils/filesystem.lua:2: module 'nfd' not found:
no field package.preload['nfd']no file 'selene/selene/lib/nfd.lua'
no file 'selene/selene/lib/nfd/init.lua'
no file 'olympUI/nfd.lua'
no file 'olympUI/nfd/init.lua'
no file 'nfd/nfd.lua'
no file 'nfd.lua'
no file 'nfd/init.lua'
no 'nfd' in LOVE game directories.
no file 'nfd' in LOVE paths.
no file './nfd.lua'
no file '/usr/local/share/luajit-2.1.0-beta3/nfd.lua'
no file '/usr/local/share/lua/5.1/nfd.lua'
no file '/usr/local/share/lua/5.1/nfd/init.lua'
no file './nfd.so'
no file '/usr/local/lib/lua/5.1/nfd.so'
no file '/usr/local/lib/lua/5.1/loadall.so'
no file '/Users/shane/Downloads/nfd.so'


Traceback

[C]: in function 'require'
utils/filesystem.lua:2: in function 'f'
selene/selene/wrappers/searcher/love2d/searcher.lua:27: in function <selene/selene/wrappers/searcher/love2d/searcher.lua:22>
[C]: in function 'require'
utils/utils.lua:2: in function 'f'
selene/selene/wrappers/searcher/love2d/searcher.lua:27: in function <selene/selene/wrappers/searcher/love2d/searcher.lua:22>
[C]: in function 'require'
meta.lua:1: in function 'f'
selene/selene/wrappers/searcher/love2d/searcher.lua:27: in function <selene/selene/wrappers/searcher/love2d/searcher.lua:22>
[C]: in function 'require'
selene_main.lua:3: in function 'f'
selene/selene/wrappers/searcher/love2d/searcher.lua:27: in function <selene/selene/wrappers/searcher/love2d/searcher.lua:22>
[C]: in function 'require'
main.lua:23: in function 'load'
main.lua:33: in function <main.lua:31>
[C]: in function 'xpcall'
[C]: in function 'xpcall'
```

**Comment by swoolcock** [comment]
Duplicate of #2 I guess.

**Comment by swoolcock** [comment]
FYI: I fixed this by downgrading system Lua to 5.1 and copying nfd.so from Olympus into `/usr/local/lib/lua/5.1`

**Comment by Vexatos** [comment]
Fixed in next build after more testing with @swoolcock. Thanks for testing this.
